packages/SwingSet/test/test-vpid-liveslots.js

```diff
@@ -300,52 +303,24 @@ async function doVatResolveCase23(t, which, mode) {
   // now it sends three() with the promise. For now, we expect the same vpid
```

Comment:
I think this comment is stale: once promises are retired, we expect a *new* vpid in the post-resolution messages